client/app/views/contacts/merge.coffee

```diff
@@ -0,0 +1,72 @@
+t = require 'lib/i18n'
+
+module.exports = class MergeView extends Mn.ItemView
+
+    template: require 'views/templates/contacts/merge'
+
+    tagName: 'section'
+
+    className: 'merge'
+
+    attributes:
+        role: 'alertdialog'
+
+
+    behaviors: ->
+        Navigator: {}
+        Dialog:    {}
+        Form:      {}
+
+    ui:
+        submit:   '[type=submit]'
+        cancel:   '.cancel'
+        inputs:   'input'
+
+
+    events:
+        'click @ui.cancel': 'onClose'
+
+
+    initialize: ->
+        # TODO : howto marionette fashion?
+        @listenTo @, 'form:submit', @onSubmit.bind @
```

Comment:
this `listenTo` is useless : Marionette allows you to write a simple `onFormSubmit` method that'll be called automagically when `form:submit` event is trigger by itself :blush: